modular_skyrat/master_files/code/modules/surgery/organs/tongue.dm

```diff
@@ -0,0 +1,23 @@
+/obj/item/organ/tongue/abductor/handle_speech(datum/source, list/speech_args)
+	//Hacks
+	var/message = speech_args[SPEECH_MESSAGE]
+
+	if(length(message) > 1)
```

Comment:
honestly, what is the reason to not have this not early return, ya know?
so like: if(length(message) <= 1) then return, or something.

Reply:
This proc shouldn't return